descwl/analysis.py

```diff
@@ -427,18 +443,19 @@ def get_bias_tensor(self,selected,covariance):
 
         return bias_tensor
 
-    def get_matrices(self,selected):
+    def get_matrices(self,selected, get_cond_num=False, equilibrate=False):
         """Return matrices derived the from Fisher-matrix images for a set of sources.
 
-        If the Fisher matrix is not invertible or any variances are <= 0, we will drop
+        If the Fisher matrix is still not invertible or any variances are <= 0, we will drop
```

Comment:
still after what? (I guess after trying to equilibrate or the pseudo-inverse?)

Reply:
Thanks Javier for the catch, I made the comment clearer in what I just pushed.